src/pages/Provider/Provider.stories.js

```diff
@@ -0,0 +1,50 @@
+import ProviderLayout from './Provider.layout.vue';
+import ProviderDoc from './Provider.documentation.mdx';
+import { providers } from '../Providers/mock';
+import { providerProducts } from './components/ProviderProducts/mock';
+
+export default {
+  title: 'Pages/Provider',
+  component: ProviderLayout,
+  parameters: {
+    docs: {
+      page: ProviderDoc,
+    },
+  },
+  argTypes: {
+    provider: {
+      name: 'provider',
+      type: {
+        required: true,
+      },
+      description: 'The provider',
+      control: {
+        type: null,
```

Comment:
Why the type is null?

Reply:
I don't want it to be changed.

BTW: I should remove that description haha, that didn't work